proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -0,0 +1,236 @@
+# Unlock Existentials for All Protocols
+
+* Proposal: [SE-NNNN](NNNN-unlock-existential-types-for-all-protocols.md)
+* Authors: [Anthony Latsis](https://github.com/AnthonyLatsis), [Filip Sakel](https://github.com/filip-sakel), [Suyash Srijan](https://github.com/theblixguy)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+
+
```

Comment:
Could you add links to the implementation PR and pitch as per the proposal template?